### ISSUE TITLE
Tests failures with Go 1.15

### ISSUE THREAD

**Issue by eclipseo** [zmap/zlint#461]
Using zlint 1.1.0 with Go 1.15 rc 1 on Fedora Rawhide, I get these errors:

```
ok  	github.com/zmap/zlint	0.220s
github.com/zmap/zlint/lints
asn1: structure error: tags don't match (6 vs {class:2 tag:0 length:6 isCompound:false}) {optional:false explicit:false application:false private:false defaultValue:<nil> tag:<nil> stringType:0 timeType:0 set:false omitEmpty:false} ObjectIdentifier @2
--- FAIL: TestEtsiTypeAsQcStmt (0.00s)
    lint_qcstatem_etsi_type_as_statem_test.go:36: ../testlint/testCerts/QcStmtEtsiTaggedValueCert20.pem: expected error, got NA
EtsiQcsQcCompliance QCStatement must not contain a statementInfo
asn1: structure error: sequence tag mismatch
--- FAIL: TestEtsiQcCompliance (0.00s)
    lint_qcstatem_qccompliance_valid_test.go:37: ../testlint/testCerts/QcStmtEtsiQcComplWithNonEmptyStmtInfoCert19.pem: expected error, got NA
    lint_qcstatem_qccompliance_valid_test.go:37: ../testlint/testCerts/QcStmtEtsiWrongEncodingCert01.pem: expected error, got NA
asn1: structure error: tags don't match (16 vs {class:0 tag:6 length:7 isCompound:false}) {optional:false explicit:false application:false private:false defaultValue:<nil> tag:<nil> stringType:0 timeType:0 set:false omitEmpty:false}  @2
asn1: syntax error: sequence truncated
--- FAIL: TestEtsiQcType (0.00s)
    lint_qcstatem_qctype_valid_test.go:36: ../testlint/testCerts/QcStmtEtsiMissingSeqForQcTypesCert18.pem: expected error, got NA
    lint_qcstatem_qctype_valid_test.go:36: ../testlint/testCerts/QcStmtEtsiMissingOidCert09.pem: expected error, got NA
FAIL
exit status 1
FAIL	github.com/zmap/zlint/lints	0.207s
```

**Comment by cpu** [comment]
Thanks for reporting. We'll take a look at this when the project switches to Go 1.15. If you feel like contributing a PR in the meantime it would be most appreciated!

**Comment by cpu** [comment]
With Go 1.15.5 I'm only able to reproduce **one** of the failures reported:

```
~/go/src/github.com/zmap/zlint$ go version
go version go1.15.5 linux/amd64

~/go/src/github.com/zmap/zlint$ go test ./...
ok  	github.com/zmap/zlint/v2	(cached)
?   	github.com/zmap/zlint/v2/cmd/zlint	[no test files]
?   	github.com/zmap/zlint/v2/cmd/zlint-gtld-update	[no test files]
?   	github.com/zmap/zlint/v2/formattedoutput	[no test files]
ok  	github.com/zmap/zlint/v2/lint	(cached)
ok  	github.com/zmap/zlint/v2/lints	0.017s
ok  	github.com/zmap/zlint/v2/lints/apple	(cached)
ok  	github.com/zmap/zlint/v2/lints/cabf_br	(cached)
ok  	github.com/zmap/zlint/v2/lints/cabf_ev	(cached)
ok  	github.com/zmap/zlint/v2/lints/community	(cached)
--- FAIL: TestEtsiQcCompliance (0.00s)
panic: Failed to parse x509 test certificate from "../../testdata/QcStmtEtsiQcComplWithNonEmptyStmtInfoCert19.pem" - "EtsiQcsQcCompliance QCStatement must not contain a statementInfo" Does a unit test have a buggy test cert file?
 [recovered]
	panic: Failed to parse x509 test certificate from "../../testdata/QcStmtEtsiQcComplWithNonEmptyStmtInfoCert19.pem" - "EtsiQcsQcCompliance QCStatement must not contain a statementInfo" Does a unit test have a buggy test cert file?


<SNIPPED>
FAIL	github.com/zmap/zlint/v2/lints/etsi	0.019s
ok  	github.com/zmap/zlint/v2/lints/mozilla	(cached)
ok  	github.com/zmap/zlint/v2/lints/rfc	(cached)
?   	github.com/zmap/zlint/v2/test	[no test files]
ok  	github.com/zmap/zlint/v2/util	(cached)
FAIL

```

[Dropping the problematic testcase](https://github.com/cpu/zlint/commit/b83ed3330a4a0070ef042f8dde8cfae1129f3e89) is sufficient to [unblock a Go 1.15.x migration](https://github.com/cpu/zlint/runs/1468254388?check_suite_focus=true).

I have an equivalent version bump [open upstream with ZCrypto](https://github.com/zmap/zcrypto/pull/235), and the Github actions work my [ZLint Go 1.15.x branch](https://github.com/cpu/zlint/tree/cpu-go-1.15) is built on is still in-review as well. Once those land I'll put up a PR to fix this test case in master and update to Go 1.15.x. Ideally before the [v3.0.0 release](https://github.com/zmap/zlint/issues/503) tag is cut but no promises.

**Comment by cpu** [comment]
Master is building/testing under Go 1.15 without error now. Please feel free to open a new issue if you see more failures in Rawhide or elsewhere that were missed.
